packages/jwt-middleware/index.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+
+const Assert = require('assert');
+const ExpressJwt = require('express-jwt');
+const jws = require('jws');
+const JwksRsa = require('jwks-rsa');
+
+const WT_AUTH_EXEC = 'wt-authorize-execution';
+
+module.exports = () => {
+
+    return function(req, res, next) {
+
+        const ctx = req.webtaskContext;
+    
+        const iss = ctx.meta['iss'];
+        const aud = ctx.meta['aud'];
+
+        // if wt-authorize-execution is set to true (non-zero) then proceed with authn and authz
+
+        if (ctx.meta[WT_AUTH_EXEC] && ctx.meta[WT_AUTH_EXEC] !== "0"){
+
+            if (req.headers['authorization']) {
+
+                const match = (req.headers['authorization'] || '')
+                .trim()
+                .match(/^bearer (.+)$/i);
+
+                let jwt;
+
+                try {
+                    jwt = jws.decode(match[1]);
+                    if (!jwt) throw new Error();
+                } catch(e){
+                    const error = new Error('Invalid authorization header');
+                    error.statusCode = 401;
+                    return next(error);
+                }
+
+                // verify claims
+
+                if (
+                    jwt.payload.scope.indexOf("wt:admin") === -1 
+                    && jwt.payload.scope.indexOf(`wt:owner:${req.x_wt.container}`) === -1
+                    && jwt.payload.scope.indexOf(ctx.meta['wt-execution-scope']) === -1
```

Comment:
`scope` in JWT's payload is a single string with whitespace delimited scopes. Doing the check you do above will result in `"wt:administrators wt:foobar"` matching the `.indexOf("wt:admin")`. I believe the safest check would be to first `.split(' ')` and then do .indexOf on the resulting array.